website/content/docs/upgrading/deduplication/index.mdx

```diff
@@ -0,0 +1,221 @@
+---
+layout: docs
+page_title: Resolve duplicate identities
+description: >-
+   Find duplicate identities in your Vault cluster and safely resolve them.
+---
+
+# Find and resolve duplicate Vault identities
+
+Bugs in Vault versions before 1.19 might lead to duplicate identities for
+entities, aliases, and groups. Duplicate identities in the persistent storage
```

Comment:
[Val's suggestion](https://github.com/hashicorp/vault/pull/29308#discussion_r1905964750) from other PR was to link this first mention of identities or the specific types to the identity docs (presumably https://developer.hashicorp.com/vault/docs/concepts/identity). I think it's a good idea but will leave it to you @schavis based on our normal conventions around things like that.